docs/conf.py

```diff
@@ -58,7 +58,7 @@ def setup(app):
     'sphinx.ext.autosummary',
     'sphinx.ext.graphviz',
     'matplotlib.sphinxext.plot_directive',
-    'numpydoc',
+    'sphinxcontrib.napoleon',
```

Comment:
@Centrus007 due to [this bug](https://stackoverflow.com/a/43237890/3912576) I have changed from `numpydoc` to `sphinxcontrib.napoleon`.